lib/CMakeLists.txt

```
@@ -33,7 +32,11 @@ set(LinkedLibs
 )
 
 if(TORCH_MLIR_ENABLE_STABLEHLO)
-list(APPEND LinkedLibs StablehloLinalgTransforms StablehloPasses)
+  list(APPEND LinkedLibs StablehloLinalgTransforms StablehloPasses)
+endif()
+
+if (TORCH_MLIR_ENABLE_TOSA)
```

nit: CMake style (fine to ignore if this is the only change)
```suggestion
if(TORCH_MLIR_ENABLE_TOSA)
```